# HG changeset patch
# User Marco Atzeri <marco_atzeri@yahoo.it>
# Date 1242923496 -7200
#      Thu May 21 18:31:36 2009 +0200
# Node ID 60bbc66bb0e202b1145849a1292ab9320fd793a6
# Parent  f27a8c07f0b2e7736ad3a3bdf4faf1dcd7b6d340
Correct my previous mistake for cygwin SHLBIN and SHLBINPRE

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,11 @@
+2009-05-21  Marco Atzeri  <marco_atzeri@yahoo.it>
+        * configure.in: Correct my mistake in cygwin SHLBINPRE SHLBIN
+
 2008-05-21  Michael Goffioul <michael.goffioul@gmail.com>
 
 	* aclocal.m4 (AC_CHECK_QHULL_OK): Do not use -lm under
 	Windows systems.
 	* configure.in: Add -luser32 to LIBS under Windows.
 
 	* configure.in (SCRIPTS_EXE_SUFFIX): New substituted
 	variable. Define to ".exe" for MSVC compilation.
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -1249,22 +1249,23 @@ case "$canonical_host_type" in
     library_path_var=DYLD_LIBRARY_PATH	
   ;;
   *-*-cygwin*)
     CPICFLAG=
     CXXPICFLAG=
     FPICFLAG=
     LIBPRE=lib
     SHLPRE=cyg
+    SHLBINPRE=cyg
     SHLEXT=dll
     SHLLIB=dll.a
-    SHLBIN=a    
+    SHLBIN=dll    
     DL_LDFLAGS="-shared -Wl,--export-all-symbols -Wl,--enable-auto-import -Wl,--enable-runtime-pseudo-reloc"
     SH_LDFLAGS="-shared -Wl,--export-all-symbols -Wl,--enable-auto-import -Wl,--enable-auto-image-base"
-    SONAME_FLAGS='-Wl,--out-implib=$(patsubst $(SHLPRE)%,$(SHLBINPRE)%,$@).a'
+    SONAME_FLAGS='-Wl,--out-implib=$(patsubst $(SHLPRE)%,$(LIBPRE)%,$@).a'
   ;;
   *-*-mingw*)
     CPICFLAG=
     CXXPICFLAG=
     FPICFLAG=
     SHLEXT=dll
     SHLLIB=dll.a
     SHLBIN=dll
